app/users/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 5.0.2 on 2024-03-19 08:51
+# Generated by Django 5.0.2 on 2024-04-05 14:13
```

Comment:
A nitpick, but while we're reviewing, can you please take out this part, since it doesn't do anything?